.travis.yml

```diff
@@ -0,0 +1,32 @@
+linux: &linux
+   os: linux
+   dist: xenial
+   language: minimal
+   sudo: required
+   services:
+     - docker
+osx: &osx
+   os: osx
+   language: generic
+osx: &windows
+   os: windows
+   language: cpp
+matrix:
+   include:
+      - <<: *linux
+        env: DOCKER_IMAGE=lasote/conangcc8
```

Comment:
Docker image are pulled from the [bincrafters SFML configuration](https://github.com/bincrafters/conan-sfml)